src/pages/configeditor/download_docker.md

```diff
@@ -19,29 +19,29 @@ subcollection: wh-acd
 
 <!-- # Overview -->
 
-The following will walk through the steps necessary to set up and configure a system with the Annotator for Clinical Data Configuration Editor (ACD Configuration Editor).  This sets up a local copy of the Configuration Editor in an "all-in-one" single system that can be used to create, edit and preview Annotator for Clinical Data (ACD) cartridges.  The Configuration Editor can also deploy and test against a remote ACD instance that you have access to.  The diagram below shows the flows in and out of the server you'll be setting up.  Ensure any firewalls are open to allow this traffic.
+The following will walk through the steps necessary to set up and configure a system with the Annotator for Clinical Data Configuration Editor (ACD Configuration Editor) using Docker.  This sets up a local copy of the Configuration Editor in an "all-in-one" single system that can be used to create, edit and preview Annotator for Clinical Data (ACD) cartridges.  The Configuration Editor can also deploy and test against a remote ACD instance that you have access to.  The diagram below shows the flows in and out of the server you'll be setting up.  Ensure any firewalls are open to allow this traffic.
 
 ![Configuration Editor on Docker](../../images/ACDCE_on_docker.png)
 
 ## Prerequisites
 
 The ACD Configuration Editor setup we use here should be installed on a dedicated system with these requirements:
 
-### System Requirements
+### System requirements
 
 - Hardware: Processor Architecture: x86_64 / amd64 ; vCPU: 4+ ; Memory: 64GB+ ; Storage: 200GB+
 - OS: CentOS 7/8 (RHEL should also work; Ubuntu may work but hasn't been tested)
 
   **NOTE:** The disk where Docker stores its images and where ACD Configuration Editor is installed needs to have 200GB of storage free.  Note: If you are using IBM Cloud with a VM, it may dafault to a 25GB base disk which will not fit the images and the Configuration Editor, so increase that or add a second larger disk and install Docker and its registry to that along with the ACD Configuration Editor.
 
-### Software Requirements (refer to the installation and configuration instructions below)
+### Software requirements (refer to the Installation and configuration instructions below)
```

Comment:
If you are capitalizing Installation because it's referencing a section header to follow, it seems like it should be a link to that specific header; otherwise, I think I would leave that one as lowercase "installation".